modules/installation-configuring-nutanix-failure-domains.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc
+// * installing/installing_nutanix/installing-restricted-networks-nutanix-installer-provisioned.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installation-configuring-nutanix-failure-domains_{context}"]
+= Configuring failure domains
+
+Failure domains improve the fault tolerance of an {product-title} cluster by distributing control plane and compute machines across multiple Nutanix Prism Elements (clusters).
+
+[TIP]
+====
+It is recommended that three failure domains be configured to ensure high-availability.
+====
+
+.Prerequisites
+
+* You have an installation configuration file (`install-config.yaml`).
+
+.Procedure
+
+. Edit the `install-config.yaml` file and add the following stanza to configure the first failure domain:
++
+[source,yaml]
+----
+# ...
+platform:
+  nutanix:
+    failureDomains:
+    - name: <failure_domain_name>
+      prismElement:
+        name: <prism_element_name>
+        uuid: <prism_element_uuid>
+      subnetUUIDs:
+      - <network_uuid>
+# ...
+----
++
+where:
+
+`<failure_domain_name>`:: Specifies a unique name for the failure domain. The name is limited to 64 or fewer characters, which can include lower-case letters, digits, and a dash (`-`). The dash cannot be in the leading or ending position of the name.
+`<prism_element_name>`:: Optional. Specifies the name of the Prism Element.
+`<prism_element_uuid`>:: Specifies the UUID of the Prism Element.
+`<network_uuid`>:: Specifies the UUID of the Prism Element subnet object. The subnet's IP address prefix (CIDR) should contain the virtual IP addresses that the {product-title} cluster uses. Only one subnet per failure domain (Prism Element) in an {product-title} cluster is supported.
+
+. As required, configure additional failure domains.
+. Do one of the following:
```

Comment:
```suggestion

```

Reply:
Yes. We do need both steps here. After you configure the first failure domain, you configure subsequent failure domains. "As required, configure additional failure domains".

After which, you either add the failure domains to the default machine configuration or you can add specific failure domains to the control plane and compute machine pools.

Reply:
@mjpytlak Sorry. I wasn't sure how to present this. I am thinking we don't need these two steps. 

```
. As required, configure additional failure domains.
. Do one of the following:
```
Maybe combine into one?

`. As required, configure additional failure domains by doing the following:`

Reply:
@mburke5678 These are, in fact, two distinct steps that should not be combined. I am happy to jump on a call with you to explain. But as the procedure stands, it is technically accurate and technically complete.

Reply:
Wonder if something describing why they do this would help? Like:
```suggestion
. To use the configured failure domains, do one of the following:
```

I have no idea if that's the right thing to say, but just to illustrate the concept

Reply:
Thanks @jeana-redhat I went with "To distribute control plane and compute machines across the failure domains, do one of the following:"

This is language we use elsewhere in the docs.